org-cyf-piscine/content/piscine/projects/music-data/index.md

```diff
@@ -0,0 +1,72 @@
+# Music data project
+
+One of the most common tasks in software is to take some data, process it in some way, and present it usefully to a user.
+
+For this project, we have supplied a collection of data for you to analyse. Your task is to write code which processes it and shows useful conclusions to a user.
+
+Your task is to make a small HTML + CSS + JavaScript frontend which displays the answers to several questions, which are listed below.
```

Comment:
I think the "task" phrasing here potentially undermines the point below about focusing on data processing. Perhaps something like this would help?

```suggestion
You should make a small HTML + CSS + JavaScript frontend which displays the answers to several questions, which are listed below.
```